plan/new_plans.go

```diff
@@ -105,6 +105,11 @@ type NewTableScan struct {
 	LimitCount *int64
 }
 
+// Truncate truncates child's rows.
+type Truncate struct {
```

Comment:
Truncate conflicts with "truncate table" statement, rename it to "Trim' is better.